platforms/hyperledger-indy/configuration/roles/create/helm_component/crypto/templates/crypto-generate.tpl

```diff
@@ -25,6 +25,7 @@ spec:
       address: {{ vault.url }}
       keyPath: {{ vault_path }}
       identity: {{ identity_name }}
+      auth_path: kubernetes-{{ organization }}-admin-auth
```

Comment:
Should be present also here

https://github.com/hyperledger-labs/blockchain-automation-framework/blob/develop/platforms/hyperledger-indy/charts/indy-key-mgmt/values.yaml